CHANGES.rst

```diff
@@ -106,7 +106,14 @@ Version 7.0
   path so that changing working directories does not harm it.
 - Force stdout/stderr writable.  This works around issues with badly patched
   standard streams like those from jupyter.
+- Add support for auto-completion documentation. See (`#866`_).
```

Comment:
This is already mentioned earlier in the log. Keep all the completion logs together.